bounties/php/CMS/1/README.md

```diff
@@ -0,0 +1,17 @@
+## Discription
```

Comment:
Spelling mistake and please make this a single `#`.

Cheers! 🍰

Reply:
done <3